tests/test_env.py

```diff
@@ -58,6 +60,25 @@ def test_fail_to_get_script_path(mocker):
     assert get_path.call_count == 1
 
 
+@pytest.mark.skipif(sys.version_info[0] == 2, reason='venv module used on Python 3 only')
+@pytest.mark.skipif(IS_PYPY3, reason='PyPy3 uses get path to create and provision venv')
+def test_fail_to_get_purepath(mocker):
+    sysconfig_get_path = sysconfig.get_path
+
+    def mock_sysconfig_get_path(path, *args, **kwargs):
+        if path == 'purelib':
+            return ''
+        else:
+            return sysconfig_get_path(path, *args, **kwargs)
```

Comment:
You don't need the else branch here, and I think you can even make it a one-liner 🤔

Reply:
eh, I don't think it really matters and 'implicit' `else`s throw me off, personally.

Reply:
It's also bad Typing, you'd be implicitly returning `str | None` instead of `str`. I thought about doing `return '' if path == 'purelib' else sysconfig_get_path(path, *args, **kwargs)` but decided against it, this looked more readable I thought.